editor/filesystem_dock.cpp

```diff
@@ -2301,19 +2253,18 @@ void FileSystemDock::_file_list_rmb_pressed(const Vector2 &p_pos) {
 	file_list_popup->add_item(TTR("New Scene..."), FILE_NEW_SCENE);
 	file_list_popup->add_item(TTR("New Script..."), FILE_NEW_SCRIPT);
 	file_list_popup->add_item(TTR("New Resource..."), FILE_NEW_RESOURCE);
-	file_list_popup->add_item(TTR("Show in File Manager"), FILE_SHOW_IN_EXPLORER);
+	file_list_popup->add_separator();
+	file_list_popup->add_item(TTR("Open in File Manager"), FILE_SHOW_IN_EXPLORER);
```

Comment:
"Open" makes sense for folders, but not that much for files, as we are not really "opening" them. For both, I think "show" makes more sense, but we might need other opinions/_favor on that.

Reply:
This popup appears when clicking in a empty area of the folder view in split mode.

Reply:
Oh if it is not used on files it makes sense then.